overlord/hookstate/ctlcmd/start.go

```diff
@@ -27,10 +27,13 @@ import (
 
 var (
 	shortStartHelp = i18n.G("Start services")
+	longStartHelp  = i18n.G(`
+The start command starts given services of the snap. If executed from the
```

Comment:
`The start command starts the given ...` (I think there is a missing `the` here).